app/[locale]/academics/scholarships/page.tsx

```diff
@@ -1,9 +1,235 @@
-import WorkInProgress from '~/components/work-in-progress';
+import Link from 'next/link';
+import { ReactNode } from 'react';
 
-export default function Scholarships({
+import Heading from '~/components/heading';
+import ImageHeader from '~/components/image-header';
+import { buttonVariants } from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { cn } from '~/lib/utils';
+
+export default async function Scholarships({
   params: { locale },
 }: {
   params: { locale: string };
 }) {
-  return <WorkInProgress locale={locale} />;
+  const text = (await getTranslations(locale)).Scholarships;
+
+  return (
+    <>
+      <ImageHeader
+        title={'Scholarships'}
+        headings={[
+          {
+            // National Scholarship Portal
+            label: 'NSP',
+            href: '#nsp',
+          },
+          {
+            // Har-Chhatravratti Scholarship
+            label: 'HCS (Haryana)',
+            href: '#hcs',
+          },
+          {
+            // Single Sign On (SSO) Scholarship
+            label: 'SSO (Rajasthan)',
+            href: '#sso',
+          },
+          {
+            // Mukhyamantri Medhavi Vidhyarthi Yojna (MMVY) (Madhya Pradesh)
+            label: 'MMVY (Madhya Pradesh)',
+            href: '#mmvy',
+          },
+          {
+            // Uttar Pradesh Scholarship
+            label: 'UPS',
+            href: '#ups',
+          },
+          {
+            // Post Matric Bihar Scholarship
+            label: 'PMBS',
+            href: '#pmbs',
+          },
+          {
+            // Prime Ministers Special Scholarship Scheme
+            label: 'PMSSS to J&K Students',
+            href: '#pmsss',
+          },
+        ]}
+        src={`https://nitkkr.ac.in/wp-content/uploads/2022/12/IMG_0496-scaled.jpg`}
+      />
+      <main className="container">
+        <Heading glyphDirection={'dual'} heading={'h1'} text={text.NSP} />
+        <ScholarshipDisplay
+          about={
+            "The National Scholarships Portal (NSP) is a comprehensive platform designed to streamline scholarship services for students. It encompasses various stages of scholarship processes, including student application, receipt, processing, sanction, and disbursal. NSP operates as a Mission Mode Project (MMP) under the National e-Governance Plan (NeGP), aligning with the government's digital initiatives."
+          }
+          description={
+            'The NSP portal hosts a range of scholarship schemes catering to various categories such as General, OBC, SC, ST, DNT, etc. Some notable schemes include the Top Class Education Scheme for SC Students and the PM Yasasvi Central Sector Scheme of Top Class Education in College for OBC, EBC, and DNT students. These schemes are initiated by the Union Government, State Governments, and Union Territories, aiming to support students financially and promote education accessibility.'
+          }
+          readMore=" https://scholarships.gov.in/"
```

Comment:
Please verify such issues before pushing.
```suggestion
          readMore="https://scholarships.gov.in/"
```